test/e2e/e2e_test.go

```diff
@@ -1197,6 +1199,176 @@ var _ = Describe("e2e non-vxlan external gateway and update validation", func()
 	})
 })
 
+// Validate the egress firewall policies by applying a policy and verify
+// that both explicitly allowed traffic and implicitly denied traffic
+// is properly handled as defined in the crd configuration in the test.
+var _ = Describe("e2e egress firewall policy validation", func() {
+	const (
+		svcname                string = "egress-firewall-policy"
+		exFWPermitTcpDnsDest   string = "8.8.8.8"
+		exFWDenyTcpDnsDest     string = "8.8.4.4"
+		exFWPermitTcpWwwDest   string = "1.1.1.1"
+		ovnNs                  string = "ovn-kubernetes"
+		ovnWorkerNode          string = "ovn-worker"
+		ovnHaWorkerNode        string = "ovn-control-plane2"
+		ovnContainer           string = "ovnkube-node"
+		ovnControlNode         string = "ovn-control-plane"
+		egressFirewallYamlFile string = "egress-fw.yml"
+		testTimeout            string = "5"
+		retryInterval                 = 1 * time.Second
+		retryTimeout                  = 30 * time.Second
+	)
+	var (
+		haMode        bool
+		ciNetworkFlag string
+		ovnNsFlag     = fmt.Sprintf("--namespace=%s", ovnNs)
+	)
+	f := framework.NewDefaultFramework(svcname)
+
+	// Determine what mode the CI is running in and get relevant endpoint information for the tests
+	BeforeEach(func() {
```

Comment:
nit ginkgo.BeforeEach

Reply:
Thanks, I can never keep up with whether dot imports are in or out of fashion :)